### ISSUE TITLE
[Bug]: namespace includes common variable names

### ISSUE THREAD

**Issue by andgoldschmidt** [kestrelquantum/QuantumCollocation.jl#132]
### What happened?

In `_integrators.jl`,

```Julia
export state
export controls
export timestep
export comps
export dim
```

### Version

stable release

### What does this bug affect?

- [ ] quantum system construction
- [ ] problem setup
- [ ] problem solution
- [ ] problem performance
- [ ] solution analysis
- [ ] plotting
- [ ] documentation
- [ ] tests
- [ ] other (please specify below)

**Comment by aarontrowbridge** [comment]
these should all be prefixed with `get_` probably

**Comment by andgoldschmidt** [comment]
closed by #149
